### PR TITLE
dev-util/hip-5.0.2: fix HIP_PATH in hipvars.pm

### PR DESCRIPTION
**Pull request by littlewu2508**
Bug: https://github.com/gentoo/gentoo/pull/25536#issuecomment-1153136503
Bug: https://github.com/justxi/rocm/issues/8#issuecomment-1152988796
Suggested-by: Paul Preney <paul@preney.ca>
Signed-off-by: Yiyang Wu <xgreenlandforwyy@gmail.com>

### CODE REVIEW

File: dev-util/hip/hip-5.0.2.ebuild [hunk 1]
```diff
@@ -79,7 +79,7 @@ src_prepare() {
 	hprefixify $(grep -rl --exclude-dir=build/ "/usr" "${S}")
 	hprefixify $(grep -rl --exclude-dir=build/ "/usr" "${HIP_S}")
 
-	cp "$(prefixify_ro "${FILESDIR}"/hipvars.pm)" bin/ || die "failed to replace hipvars.pm"
+	cp "$(prefixify_ro "${FILESDIR}"/hipvars-5.0.2.pm)" bin/hipvars.pm || die "failed to replace hipvars.pm"
```
Comment (heroxbd) [inline dev-util/hip/hip-5.0.2.ebuild:82]: use `dobin` and leave out `|| die`

**Review by heroxbd — CHANGES_REQUESTED** [review]

**Comment by heroxbd** [comment]
This needs a revision bump, because the we want users who successfully compiled hip-5.0.2 to upgrade to this version. And roctracer etc. should depend on the new revision.

File: dev-util/hip/hip-5.0.2.ebuild (hunk 1 above)
Comment (littlewu2508) [reply dev-util/hip/hip-5.0.2.ebuild:82]: I think `dobin` is an install function while this is `src_prepare`?

**Comment by heroxbd** [comment]
littlewu2508 ***@***.***> writes:

> I think dobin is an install function while this is src_prepare?

Yes, you are correct.  I overlooked that fact that bin/ is under ${S}, sorry.

**Pull request closed without merging**